src/app/screens/Onboard/ConnectRaspiBlitz/index.tsx

```diff
@@ -101,8 +108,8 @@ export default function ConnectRaspiBlitz() {
                 id="url"
                 label="REST API host"
                 placeholder="your-node-url"
-                pattern="https://.+"
-                title="https://your-node-url:8080"
+                pattern="^[A-Za-z0-9]*\.onion$"
```

Comment:
is it always the .onion address? can we be sure about that?

Reply:
no we cannot be sure. Here it works only for nodes with a onion address.
What type of address does the user need if he wants to connect a public node?

Reply:
the public IP address.
but then let's just not add this .onion validation and leave it open.

Reply:
Ok but now we also need to update the instructions - currently they are instructing the user to use the .onion address. Can we add the other instructions to get the non-tor connection info?

Reply:
Ok I have removed the validation but I don't think any users will assume they can use clearnet with the current instructions because it says 'paste the .onion address below'. But i'll post a screenshot of the latest instructions so you can take a look.